### PR TITLE
doc: add minutes for 30 May 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2017-05-30.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+* Gibson Fahnestock 
+  * Minor Jenkins fixes (skipping platforms etc)
+  * Refactored CitGM builds to fix AIX issue
+* Michele Capra
+  * Testing ansible refactor on windows
+* Johan Bergström 
+  * merging ansible refactor
+  * minor review
+  * going to update clang version of our freebsd builders to address v8 build issues
+*Kyle/Kunal
```
Comment (kfarnung) [inline doc/meetings/build-wg-meeting-2017-05-30.md]: Missing space after the asterisk

File: doc/meetings/build-wg-meeting-2017-05-30.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+## Standup
+* Michael Dawson
+  * Starting on ansible scripts for zOS
+  * Reaching out to mac provider
+  * Working to get more benchmarking team participants
+    working with benchmarking CI jobs
+* João Reis 
+  * Addedd Ubuntu and OSX to Node-ChakraCore test job
+  * Changed the Node-ChakraCore nightlies release server to
+   Ubuntu 14.04, opened PR with Ansible changes
```
Comment (kfarnung) [inline doc/meetings/build-wg-meeting-2017-05-30.md]: nit: add a space to indent

File: doc/meetings/build-wg-meeting-2017-05-30.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    
+### downloads/cloudflare
+* Call to action to help get cloudflare coverage
+* Main work is to capture metrics from cloudflare logs versus our logs,
+  Rod: half done if anybody can help out that would be great.
+  Rod, to create an issue.
+ 
+### Solution to vendors to rsync securely [#55](https://github.com/nodejs/build/issues/55)
+* @ofrobots a better/more secure solution for downloads ?
+* Rodd: We.ve looked at it but no great answers.
```
Comment (kfarnung) [inline doc/meetings/build-wg-meeting-2017-05-30.md]: nit: We've

File: doc/meetings/build-wg-meeting-2017-05-30.md (hunk 2 above)
Comment (kunalspathak) [inline doc/meetings/build-wg-meeting-2017-05-30.md]: nit: and also `*`

**Comment by mhdawson** [comment]
Landed as 0ef7fddb3d655033b47b713885936ae8b12fcd32

**Pull request closed without merging**